pulsar-client/src/main/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImpl.java

```diff
@@ -955,7 +955,7 @@ private void doSubscribeTopicPartitions(Schema<T> schema,
                     })
                 .collect(Collectors.toList());
         } else {
-            boolean isTopicBeingSubscribedForInOtherThread = this.topics.putIfAbsent(topicName, 1) != null;
+            boolean isTopicBeingSubscribedForInOtherThread = this.topics.putIfAbsent(topicName, 0) != null;
```

Comment:
Does it make sense to create a constant for this magic value?

Reply:
Good point. Added constant